api/v1alpha1/openapi.yaml

```diff
@@ -5186,7 +5193,7 @@ components:
           type: array
           items:
             type: string
-            enum: [owner, labels, spec]
+            enum: [ owner, labels, spec ]
```

Comment:
This definition of `updatedFields` is not according to the current implementation. The enum restriction should be dropped.

Reply:
@avishayt WDYT?